orchagent/portsorch.cpp

```diff
@@ -645,6 +781,27 @@ void PortsOrch::doPortTask(Consumer &consumer)
                     }
                 }
             }
+
+            if (pvid != 0)
```

Comment:
change this part of code to be similar as the previous mtu part of code.

Reply:
ok